datalad/crawler/nodes/matches.py

```diff
@@ -176,7 +176,6 @@ def _select_and_extract(self, selector, query, data):
 
 class a_href_match(AExtractorMatch):
     """Helper to simplify matching based on URL while also extracting various tags from URL while at it
-
```

Comment:
be careful to not modify (without reason) code unrelated to your changes

here we are following numpy docstrings convention (linked in CONTRIBUTING.md) and an empty line is mandatory